templates/default/prometheus.yml.erb

```diff
@@ -9,7 +9,10 @@ global:
 alerting:
   alertmanagers:
   - static_configs:
-    - targets: ['<%= @alertmanagers %>']
+    - targets:
```

Comment:
For Prometheus `alertmanagers` configuration I think the `static_configs` is not suitable anymore. According to the documentation you can use `consul_sd_configs` the same way we do for scraping